src/components/Heading/Heading.tsx

```diff
@@ -41,7 +42,7 @@ export const headingLevelsArray = [
 export type HeadingSizes = typeof headingSizesArray[number];
 export type HeadingLevels = typeof headingLevelsArray[number];
 
-export interface HeadingProps {
+export interface HeadingProps extends ChakraHeadingProps {
```

Comment:
We use this same pattern for the `Link` component and this should be the pattern for other components if they need it. I'm not sure if we should just go ahead and do this for all components now or wait until it's needed but it's not a big blocker in consuming apps.

Reply:
If we apply this method to other components, will this open up potential problems because it will expose props that we may not want consuming apps to use?  And if that is the case, should we get over ourselves and not worry too much about it?

Reply:
Ha, yes sort of. But it's already restrictive because it's for the Chakra `Heading` component. So, for example, `alt` is not applicable here so there's some safety in there.